modules/oe_theme_content_project/oe_theme_content_project.module

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the Project content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function oe_theme_content_project_node_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  // Run only on project content type.
+  if ($entity->bundle() !== 'oe_project') {
+    return;
+  }
+
+  // Field labels are different in frontend and backend.
+  // Here we override them, per view mode.
+  if ($build['#view_mode'] === 'full') {
+    $label_overrides = [
+      'oe_project_dates' => t('Project duration'),
+      'oe_project_website' => t('Project website'),
+    ];
+    // If there is only a start date, set "Start date" as label.
+    $dates = $entity->get('oe_project_dates')->getValue();
+    if (!$dates[0]['end_value'] || $dates[0]['value'] === $dates[0]['end_value']) {
+      $label_overrides['oe_project_dates'] = t('Start date');
+    }
```

Comment:
We need to test this with Behat, using our "oe_content Behat APIs", we first create a project with only start date, then we assert this label, then we add the end date, we refresh the page, and we assert the other label. The refreshing of the page is the important part, as it will allow us to correctly test caching invalidation.

Reply:
`oe_project` form display changes have been done on [oe_content PR221](https://github.com/openeuropa/oe_content/pull/221/files). Please @ademarco can you merge it?

Reply:
Also, no Behat tests are provided yet, we should add them before moving this into review.

Reply:
Test is provided in `tests/features/project.feature`